lua/json_object_array.lua

```diff
@@ -0,0 +1,14 @@
+-- Lua script to transform string arguments into an array of dictionaries and output as JSON
+
+local cjson = require("cjson")
+
+local my_array = {}
+
+for i = 1, #arg do
+    local string = arg[i]
+    local dict = {}
+    dict[string:upper()] = string:len()
+    table.insert(my_array, dict)
```

Comment:
```suggestion
    table.insert(my_array, {
        [arg[i]:upper()] = arg[i]:len()
    })
```

Here's a way to write this more succintly